libqfieldsync/project_checker.py

```diff
@@ -439,3 +444,47 @@ def check_external_layers(
             )
 
         return None
+
+    def check_layer_has_experimental_cloud_support(
+        self, layer_source: LayerSource
+    ) -> Optional[FeedbackResult]:
+        """Check if layer has experimental cloud support"""
+        layer = layer_source.layer
+
+        # Check if layer is marked for offline editing
+        if layer_source.cloud_action != SyncAction.OFFLINE:
+            return None
+
+        # Check if layer is read-only
+        if layer.readOnly():
+            return None
+
+        # Check if layer is in a supported format
+        provider_type = layer.providerType()
+
+        # Check if it's a GeoPackage
+        if layer.dataProvider().storageType() == "GPKG":
+            return None
+
+        # Check if it's a PostgreSQL database
+        if provider_type == "postgres":
+            return None
+
+        # Check if it's a non-spatial delimited text file
+        if (
+            provider_type == "delimitedtext"
+            and layer.dataProvider().storageType() == "CSV"
+        ) or (
+            provider_type == "ogr"
+            and layer.dataProvider().storageType() == "Delimited text file"
+        ):
```

Comment:
```suggestion
        if (
            provider_type == "delimitedtext" and storage_type == "Delimited text file"
        ) or (
            provider_type == "ogr" and storage_type == "CSV"
        ):
```
I am pretty sure these two were swapped, prove me wrong.

Reply:
Reviewing now, yes indeed I swapped!.... Apologize